packages/api/tests/endpoints/test-pdrs.js

```diff
@@ -406,48 +379,13 @@ test.serial('DELETE a pdr', async (t) => {
   t.is(detail, 'Record deleted');
 });
 
-test.serial('DELETE handles the case where the PDR exists in S3 but not in DynamoDb', async (t) => {
+test.serial('DELETE handles the case where the PDR exists in PostgreSQL but not in S3', async (t) => {
   const {
-    knex,
-    pdrPgModel,
-    testPgProvider,
-    testPgCollection,
-  } = t.context;
-
-  const nonExistentPdr = fakePdrFactoryV2({
-    collectionId: constructCollectionId(testPgCollection.name, testPgCollection.version),
-    provider: testPgProvider.name,
-  });
-  const insertPgRecord = await translateApiPdrToPostgresPdr(nonExistentPdr, knex);
-  await pdrPgModel.create(knex, insertPgRecord);
-
-  const pdrName = insertPgRecord.name;
-
-  await uploadPdrToS3(
-    process.env.system_bucket,
-    pdrName,
-    'This is the PDR body'
-  );
-
-  const response = await request(app)
-    .delete(`/pdrs/${pdrName}`)
-    .set('Accept', 'application/json')
-    .set('Authorization', `Bearer ${jwtAuthToken}`)
-    .expect(200);
-
-  t.is(response.status, 200);
-
-  const parsedBody = response.body;
-  t.is(parsedBody.detail, 'Record deleted');
-});
-
-test.serial('DELETE handles the case where the PDR exists in DynamoDb but not in S3', async (t) => {
-  const {
-    originalDynamoPdr,
+    originalPgRecord,
   } = await createPdrTestRecords(t.context);
```

Comment:
It looks like `createPdrTestRecords` does stage a file to S3, so I'm not sure that this test proves what it claims to